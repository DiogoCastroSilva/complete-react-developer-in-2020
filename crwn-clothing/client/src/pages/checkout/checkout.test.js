import React from 'react';
import { shallow } from 'enzyme';

import Checkout from './checkout.component';


describe('testing Checkout component', () => {
    let wrapper;

    beforeEach(() => {
        const mockProps = {
            cartItems: [],
            total: 0
        };

        wrapper = shallow(<Checkout {...mockProps} />);
    });

    it('should render a Checkout component', () => {
        expect(wrapper).toMatchSnapshot();
    });
});