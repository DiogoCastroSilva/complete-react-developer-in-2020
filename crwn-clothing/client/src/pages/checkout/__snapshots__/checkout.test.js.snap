// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Checkout component should render a Checkout component 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <span>
        Product
      </span>
    </styled.div>
    <styled.div>
      <span>
        Description
      </span>
    </styled.div>
    <styled.div>
      <span>
        Quantity
      </span>
    </styled.div>
    <styled.div>
      <span>
        Price
      </span>
    </styled.div>
    <styled.div>
      <span>
        Remove
      </span>
    </styled.div>
  </styled.div>
  <styled.div>
    <span>
      TOTAL: $
      0
    </span>
    <styled.div>
      *Please use the following credic card for payments*
      <br />
      4242 4242 4242 4242 - Exp: 01/
      21
       - CVV: 123
    </styled.div>
    <Styled(StripeButton)
      price={0}
    />
  </styled.div>
</styled.div>
`;
